Apps/Examples/Examples/SiwftUI Examples/AnnotationsExample.swift

```diff
@@ -0,0 +1,43 @@
+import Foundation
+
+import SwiftUI
+@_spi(Experimental) import MapboxMapsSwiftUI
+
+@available(iOS 14.0, *)
+struct AnnotationsExample: View {
+    struct City {
+        var name: String
+        var location: CLLocationCoordinate2D
+    }
+
+    @State var camera = CameraState(center: .zero, zoom: 1)
+
+    var cities = [
+        City(name: "Helsinki", location: .helsinki),
+        City(name: "New York", location: .newYork),
+        City(name: "London", location: .london),
+        City(name: "Berlin", location: .berlin)
+    ]
+
+    var body: some View {
+        MapboxView(camera: $camera)
+            .annotations(cities.map {
+                var annotation = PointAnnotation(
+                    id: $0.name,
+                    coordinate: $0.location
+                )
+                annotation.image = .init(
+                    image: UIImage(named: "marker")!, name: "marker")
+                return annotation
+            })
+            .edgesIgnoringSafeArea(.all)
```

Comment:
`edgesIgnoringSafeArea` is deprecated, if we are aiming at iOS 14, let's replace with https://developer.apple.com/documentation/swiftui/view/ignoressafearea(_:edges:)